### PR TITLE
added two sourdough bread recepies

### PR DESCRIPTION
**Pull request by SirStark**
- added sourdough potato bread
- added sourdough bread with seeds and grains

### CODE REVIEW

**Comment by Socerest2** [comment]
Compress your image files. I compressed them for you, just add them in, replace, and then the check should pass.
[Sourdough-potato-bread](https://i.ibb.co/ZMq5xp8/soudough-potato-bread.webp)
[Bread With Seeds](https://i.ibb.co/nczMmpm/sourdough-bread-with-seeds-and-grains.webp)

**Comment by SirStark** [comment]
> Compress your image files. I compressed them for you, just add them in, replace, and then the check should pass.
> [Sourdough-potato-bread](https://i.ibb.co/ZMq5xp8/soudough-potato-bread.webp)
> [Bread With Seeds](https://i.ibb.co/nczMmpm/sourdough-bread-with-seeds-and-grains.webp)

Thank you very much. I have replaced the pictures.

File: src/sourdough-bread-with-seeds-and-grains.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Sourdough bread with seeds and grains
+
+![Sourdough bread with seeds and grains](pix/sourdough-bread-with-seeds-and-grains.webp)
+
+- ⏲️ Prep time: 15 min
+- 🍳 Cook time: 120 min
+
+## Ingredients
+
+**For the preparation:**
```
Comment (hallzy) [inline src/sourdough-bread-with-seeds-and-grains.md:10]:
```suggestion
### For the preparation
```

We don't really have a standard for it, but I've seen many people use the `###` for subheadings under ingredients, so let's stick with that

File: src/sourdough-bread-with-seeds-and-grains.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+- 100g grain of choice (e.g. rye, spelt and wheat)
+- 40g linseed
+- 30g sesame
+- 30g sunflower seeds
+- 600g water
+
+**For the bread:**
```
Comment (hallzy) [inline src/sourdough-bread-with-seeds-and-grains.md:18]:
```suggestion
### For the bread
```

same as above

File: src/sourdough-potato-bread.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- ⏲️ Prep time: 15 min
+- 🍳 Cook time: 45 min
+
+## Ingredients
+
+- 150g boiled potatoes in there skin
+- 350g wheat flour
+- 50g whole rye flour
+- 11g salt
```
Comment (hallzy) [inline src/sourdough-potato-bread.md:13]: Typically we don't have salt in the ingredients list either... Usually I think that is for 'taste' at the end. But this is being baked in and has a specific measurement, so maybe this is fine to leave.

File: src/sourdough-bread-with-seeds-and-grains.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- 200g water (25-35°C)
+- 200g rye flour
+- 300g wheat flour
+- 20g sea salt
+- 2tsp bread spice
+- soaked seeds and grains
+  
+## Directions
+
+1. Preparations:
```
Comment (hallzy) [inline src/sourdough-bread-with-seeds-and-grains.md:30]: Was `Preparations` left here by accident? There isn't a separation so it looks like all the directions are `Preparation`.

If you do keep it and add another heading, I'd suggest similar to above and use:

```
### Preparations
```

**Review by hallzy — CHANGES_REQUESTED** [review]

File: src/sourdough-bread-with-seeds-and-grains.md (hunk 1 above)
Comment (SirStark) [reply src/sourdough-bread-with-seeds-and-grains.md:10]: Yeah, we should define that somewhere. I looked into some recipes and did not find one with subsections. I will change it to the `###` notation.

File: src/sourdough-potato-bread.md (hunk 3 above)
Comment (SirStark) [reply src/sourdough-potato-bread.md:13]: I hope that it is okay in this case as you described. In this case the amount of salt is quite important. Also this way I can write `mix all ingredients together` without mention the amount of salt in the directions section.

File: src/sourdough-bread-with-seeds-and-grains.md (hunk 4 above)
Comment (SirStark) [reply src/sourdough-bread-with-seeds-and-grains.md:30]: That was by accident. I will change it to subsections similar to ingredients

**Comment by Socerest2** [comment]
> > Compress your image files. I compressed them for you, just add them in, replace, and then the check should pass.
> > [Sourdough-potato-bread](https://i.ibb.co/ZMq5xp8/soudough-potato-bread.webp)
> > [Bread With Seeds](https://i.ibb.co/nczMmpm/sourdough-bread-with-seeds-and-grains.webp)
> 
> Thank you very much. I have replaced the pictures.

No problem, just try to compress them to under 100kb next time. This is the compressor I [use](https://www.compresss.com/compress-webp.html#)

**Comment by SirStark** [comment]
> > > Compress your image files. I compressed them for you, just add them in, replace, and then the check should pass.
> > > [Sourdough-potato-bread](https://i.ibb.co/ZMq5xp8/soudough-potato-bread.webp)
> > > [Bread With Seeds](https://i.ibb.co/nczMmpm/sourdough-bread-with-seeds-and-grains.webp)
> > 
> > 
> > Thank you very much. I have replaced the pictures.
> 
> No problem, just try to compress them to under 100kb next time. This is the compressor I [use](https://www.compresss.com/compress-webp.html#)

the page was bookmarked immediately 😇

File: src/sourdough-bread-with-seeds-and-grains.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+2. Stir all ingredients for the bread and knead them well.
+3. Let the dough rest for 5 hours in warm (25-30°C) conditions.
+4. Preheat the oven to 250°C.
+5. Bake for up to 20 minute with steam.
+6. Release the steam and turn the heat down to 150°C.
+7. Bake for another 50-60 minutes
+
+## Contribution
+
+SirStark
```
Comment (Socerest2) [inline src/sourdough-bread-with-seeds-and-grains.md:49]:
```suggestion
- SirStark
```
Fixed Credit Format

**Review by Socerest2 — CHANGES_REQUESTED** [review]

**Review by Socerest2 — APPROVED** [review]
Looks good!

**Pull request merged**